pyproject.toml

```diff
@@ -8,6 +8,7 @@ exclude_rules = [
     "references.qualification",
     "structure.unused_cte",
     "structure.column_order",
+    "structure.subquery",
```

Comment:
Sqlfluff doesn't like subqueries within the `JOIN` parts of a query. I'm open to enforcing that, but [this file](https://github.com/NYCPlanning/data-engineering/blob/5e93eab1aae7f8c536ab5cc9cf622aad1ed4c437/products/cpdb/sql/analysis/projects_dollars_mapped_categorized_sponsor.sql) (and another similar one) were violating. The joins could be done away with by using a bunch of functions in the select clause (`SELECT COUNT(*) FILTER (WHERE typecategory = 'Fixed Asset') AS fixedassett`) but I don't know if that's particularly more readable. Could also do CTEs instead, but again, don't know if a long list of CTEs here would improve legibility